index.js

```diff
@@ -27,33 +71,45 @@ app.get('/Styles/:file', (req, res) => {
   })
 })
 
+app.get('/login', (req, res) => {
+  if (isLoggedIn(req)) {
+    res.redirect('/')
+    return
+  }
+
+  request.get({
+    url: 'http://w1131323.ferozo.com/wonderfood/Login.aspx'
+  },
+    (err, response, body) => {
+      const htmlString = render(body)
+      res.send(htmlString)
+    }
+  )
+})
+
+app.get('/logout', (req, res) => {
+  req.session = null
+  res.send('<div>Gracias, vuelva prontos! 👋</div>' +
+    '<script>' +
+    ' setTimeout(() => {' +
+    '   document.location.href = \'/\'' +
+    ' }, 1000)' +
```

Comment:
If we want the goodbye message to be read we might need to wait longer for users to read it.
:)